browser-test/src/end_to_end_enumerators.test.ts

```diff
@@ -37,7 +37,7 @@ describe('End to end enumerator test', () => {
     // Verify question preview has the default values.
     await adminQuestions.expectCommonPreviewValues({
       questionText: 'Sample question text',
-      questionHelpText: '',
+      questionHelpText: ' ',
     })
```

Comment:
**question (blocking):** Did you mean to make this change?

Reply:
Yes I did. function PreviewController.formatText() makes it being a non-breaking space if the input wasn't given.

Reply:
Interesting, wonder if that's a bug in `PreviewController`. Can you add a comment explaining why the space is there?

Reply:
The space was in the `PreviewController.formatText()` before this change, it could have been a bug or intended. I deleted the space, can add the space back later if the space was intended to be there for some reason.